TheEthicalHackerCup/Assets/RedBlueGames/TextTyper/Animation/TextAnimation.cs

```diff
@@ -0,0 +1,240 @@
+﻿namespace RedBlueGames.Tools.TextTyper
```

Comment:
If this is an external lib usage, it would be worth mentioning in the prog-report that this why this was included and the benefit of including it